src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -19,618 +19,6 @@ namespace Microsoft.CodeAnalysis.CSharp.Extensions
-                case ConstructorInitializerSyntax constructorInitializer:
-                    return constructorInitializer.ContainsInArgument(span);
-            }
-
-            return false;
-        }
-
-        public static bool ContainsInArgument(this ConstructorInitializerSyntax initializer, TextSpan textSpan)
-        {
-            if (initializer == null)
-            {
-                return false;
-            }
-
-            return initializer.ArgumentList.Arguments.Any(a => a.Span.Contains(textSpan));
-        }
-
-        public static bool ContainsInBlockBody(this BlockSyntax block, TextSpan textSpan)
-        {
-            if (block == null)
-            {
-                return false;
-            }
-
-            var blockSpan = TextSpan.FromBounds(block.OpenBraceToken.Span.End, block.CloseBraceToken.SpanStart);
-            return blockSpan.Contains(textSpan);
-        }
-
-        public static bool ContainsInExpressionBodiedMemberBody(this ArrowExpressionClauseSyntax expressionBodiedMember, TextSpan textSpan)
-        {
-            if (expressionBodiedMember == null)
-            {
-                return false;
-            }
-
-            var expressionBodiedMemberBody = TextSpan.FromBounds(expressionBodiedMember.Expression.SpanStart, expressionBodiedMember.Expression.Span.End);
-            return expressionBodiedMemberBody.Contains(textSpan);
-        }
-
-        public static IEnumerable<MemberDeclarationSyntax> GetMembers(this SyntaxNode node)
-        {
-            switch (node)
-            {
-                case CompilationUnitSyntax compilation:
-                    return compilation.Members;
-                case NamespaceDeclarationSyntax @namespace:
-                    return @namespace.Members;
-                case TypeDeclarationSyntax type:
-                    return type.Members;
-                case EnumDeclarationSyntax @enum:
-                    return @enum.Members;
-            }
-
-            return SpecializedCollections.EmptyEnumerable<MemberDeclarationSyntax>();
-        }
-
-        public static bool IsInExpressionTree(
-            this SyntaxNode node, SemanticModel semanticModel,
-            INamedTypeSymbol expressionTypeOpt, CancellationToken cancellationToken)
-        {
-            if (expressionTypeOpt != null)
-            {
-                for (var current = node; current != null; current = current.Parent)
-                {
-                    if (current.IsAnyLambda())
-                    {
-                        var typeInfo = semanticModel.GetTypeInfo(current, cancellationToken);
-                        if (expressionTypeOpt.Equals(typeInfo.ConvertedType?.OriginalDefinition))
-                        {
-                            return true;
-                        }
-                    }
-                }
-            }
-
-            return false;
-        }
-
         public static SyntaxNode WithPrependedNonIndentationTriviaFrom(
```

Comment:
literally everything moved except this... should we just move this down as well? :)   (or remove it if not used anymore).  then we can delete this file.

Reply:
Yeah, that was my initial attempt, but this method refers to `Formatter.Annotation` which is not available in CompilerExtensions layer.